cobra/cmd/license_gpl_3.go

```diff
@@ -18,23 +18,22 @@ package cmd
 func initGpl3() {
 	Licenses["gpl3"] = License{
 		Name:            "GNU General Public License 3.0",
-		PossibleMatches: []string{"gpl3", "gpl", "gnu gpl3", "gnu gpl"},
+		PossibleMatches: []string{"gpl3", "gplv3", "gpl", "gnu gpl3", "gnu gpl"},
```

Comment:
your call :) FWIW  https://www.gnu.org/licenses/gpl-3.0.html#section14 ` If the Program does not specify a version number of the GNU General Public License, you may choose any version ever published by the Free Software Foundation.`
This applies to all the version of the GPL1,2 and 3 BTW.